intellij/src/nl/tudelft/watchdog/intellij/logic/event/listeners/BreakpointListener.java

```diff
@@ -24,7 +24,7 @@
  * {@link BreakpointEventBase} is generated and given to the
  * {@link TrackingEventManager}.
  */
-public class BreakpointListener implements XBreakpointListener {
+public class BreakpointListener implements XBreakpointListener<XBreakpoint<?>> {
```

Comment:
Do we really need the additional `<XBreakpoint<?>>`?

Reply:
Yes, else it complains about missing the generics specification. E.g. `List` vs `List<Type>`

Reply:
Ok